requirements/test.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-cov
+nose
```

Comment:
You might be interested to take a look at https://github.com/scikit-image/scikit-image/issues/2424, https://github.com/scikit-image/scikit-image/pull/2468. :)

Reply:
Gah, thanks. Sorry, I had looked at a previous build for 0.13.x that failed because of nose.

The more recent builds fail because of watershed and whatnot.